src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -609,7 +609,16 @@ public function queue_import( $args = array() ) {
 
 	public function get_import_data() {
 		$aggregator = tribe( 'events-aggregator.main' );
-		return $aggregator->api( 'import' )->get( $this->meta['import_id'] );
+
+		// For now only apply this to the URL type
+		if ( 'url' === $this->type ) {
+			$data = array(
+				'start' => $this->meta['start'],
+				'end' => $this->meta['end'],
+			);
+		}
+
+		return $aggregator->api( 'import' )->get( $this->meta['import_id'], $data );
```

Comment:
_Looks_ like `$data` may not be defined at this point (depending on the result of the above conditional). Worth defining it as an empty array, some point above the `if` block?